### ISSUE TITLE
.gobject move/turn does not update client

### ISSUE THREAD

**Issue by MacWarrior** [TrinityCore/TrinityCore#3802]
http://code.google.com/p/trinitycore/issues/detail?id=447
http://code.google.com/p/trinitycore/issues/detail?id=853

What steps will reproduce the problem?
1. Spawn an object using .gobject add
2. Try turning the object with .gobject turn objectsguid
3. Try moving the object with .gobject move objectguid

What is the expected output? What do you see instead?
While the object does get updated on the database side, I expect to see the
object update itself when you move/turn it, however I get the notification
that the object was moved or turned, and the object disappears but then
reappears where it was originally spawned. I would have to teleport to a
completely different map and recall myself to see the updates client side.

**Comment by Rochet2** [comment]
What I tried some time ago was spawning a gameobject, then deleting it and spawning a new gameobject with the same guid.
Basically this _should_ have updated the postition but it appeared in the same location anyways. No teleporting to other maps or anything relocated the object until relog.
So my conclusion was that the current methods might not allow this and its not possible or there is some opcode or something that does it.
So I just used fake guids instead.

No idea if it is possible though.
